packages/navigator/src/browser/navigator-tree.ts

```diff
@@ -66,13 +66,13 @@ export namespace WorkspaceNode {
         return CompositeTreeNode.is(node) && node.name === WorkspaceNode.name;
     }
 
-    export function createRoot(): WorkspaceNode {
+    export function createRoot(multiRootName?: string): WorkspaceNode {
         return {
             id: WorkspaceNode.id,
-            name: WorkspaceNode.name,
+            name: multiRootName ? multiRootName : WorkspaceNode.name,
```

Comment:
fyi: `multiRooName || WorkspaceNode.name` is a shortcut

Reply:
nice! thank you